pkgs/tools/misc/hexdiff/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "hexdiff";
+  version = "unstable-2018-01-24";
+  src = fetchFromGitHub {
+    owner = "ahroach";
+    repo = "${pname}";
```

Don't parameterize `pname`.

```suggestion
    repo = "hexdiff";
```